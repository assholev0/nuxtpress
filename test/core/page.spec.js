import { join } from 'path';
import { page } from '../util';

describe('core/page', () => {
  test('getPage', () => {
    const lazyPage = page({
      fileName: 'page.md',
      filePath: join(__dirname, './fixtures/page.md')
    });
    expect(lazyPage).toMatchSnapshot();
  });
});
