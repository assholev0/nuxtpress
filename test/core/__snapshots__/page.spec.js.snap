// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core/page getPage 1`] = `
Object {
  "atrributes": Object {
    "category": "技术",
    "date": 2018-09-16T11:47:42.000Z,
    "edited": 2018-09-22T02:18:13.000Z,
    "tags": Array [
      "vue",
      "vuepress",
      "前端",
    ],
    "title": "开篇：你好，世界！",
  },
  "raw": Object {
    "content": "
这是一篇测试文章。
",
    "data": Object {
      "category": "技术",
      "date": 2018-09-16T11:47:42.000Z,
      "edited": 2018-09-22T02:18:13.000Z,
      "tags": Array [
        "vue",
        "vuepress",
        "前端",
      ],
      "title": "开篇：你好，世界！",
    },
    "excerpt": "",
    "isEmpty": false,
  },
}
`;
